umake/frameworks/ide.py

```diff
@@ -181,7 +181,7 @@ def post_install(self):
         comment = self.description + " (UDTC)"
         categories = "Development;IDE;"
         create_launcher(self.desktop_filename,
-                        get_application_desktop_file(name=self.name,
+                        get_application_desktop_file(name=self.description,
```

Comment:
Why changing the name by description here? I think it's fine to say that we well have idea and idea-ultimate. Any good shell will as well propose the description as a matching pattern.